tests/monotouch-test/Intents/INIntentResolutionResultTests.cs

```diff
@@ -37,34 +37,147 @@ public void INIntentResolutionResultIsAbstractTest ()
 		}
 
 		[Test]
-		public void INBooleanResolutionResultPropertyTest ()
```

Comment:
This is a big nasty diff. Can we reorder it back to the way it was so I can see the diff?